app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/Main.scala

```diff
@@ -213,11 +204,22 @@ object Main extends IOApp with HistogramStoreImplicits with LazyLogging {
 
   def stream =
     BlazeServerBuilder[IO]
+      .bindHttp(8080, "0.0.0.0")
       .withExecutionContext(blazeEC)
-      .withBanner(startupBanner)
+      .withIdleTimeout(new FiniteDuration(Config.server.idleTimeoutSeconds,
+                                          TimeUnit.SECONDS))
+      .withResponseHeaderTimeout(
+        new FiniteDuration(Config.server.responseHeaderTimeoutSeconds,
+                           TimeUnit.SECONDS))
       .withConnectorPoolSize(Config.parallelism.blazeConnectorPoolSize)
-      .bindHttp(8080, "0.0.0.0")
+      .withNio2(false)
```

Comment:
All of these appear to default to off, except the WebSockets support. That is now being explicitly disabled.